### ISSUE TITLE
Labeling SubCircuits

### ISSUE THREAD

**Issue by umesh-timalsina** [symbench/scm#183]
We need to figure out a strategy for labeling subcircuits. Simple example below for `fc.png`.
![image](https://user-images.githubusercontent.com/11476842/126184034-4512e2af-582f-4dca-ac86-db1310ceb2fa.png)

_Originally posted by @umesh-timalsina in https://github.com/symbench/scm/issues/169#issuecomment-882633430_

**Comment by umesh-timalsina** [comment]
Possible duplicate of #175.

**Comment by brollb** [comment]
One approach that would be good to try would be:
- Label the entire region as a "Circuit"
- Label the individual ports using "Port" as the label. Set the comment text to the name of the port.

My hope is that the segmentation model might be able to learn how to segment out the subcircuits. Currently, I am also training a separate model for detecting the ports. After detecting the ports, we could use OCR to hopefully determine the port names.

**Comment by brollb** [comment]
This workflow is currently blocked by #131

**Comment by brollb** [comment]
Currently, the plan is to label them as described above. In other words, the entire circuit is labeled as "Circuit" and the individual ports are "Port" with the text recorded in the comment.
